playbooks/openshift/roles/bfs/tasks/build_openshift_srpm.yml

```diff
@@ -22,13 +23,17 @@
   shell: git checkout {{ origin_version }}
   args:
     chdir: "{{ openshift_repo_path }}"
-  when: (git_log.stdout == "" and not bleeding_edge)
+  when:
+    - git_log.stdout == ""
+    - not bleeding_edge
 
 - name: tag origin version
   shell: git tag -d {{ origin_version }}
   args:
     chdir: "{{ openshift_repo_path }}"
-  when: (git_log.stdout == "" and not bleeding_edge)
+  when:
+    - git_log.stdout == "" and
```

Comment:
You didn't use parens here?

Reply:
the parens for `and` i'd normally use when we have multiple conditions which must be satisfied together, like

```
when:
   - (foo and foo is defined) or
   - bar == 'orange'
```

Reply:
and i just spotted a bug which i'm going to fix it in a bit.